public/locales/en/translation.json

```diff
@@ -244,5 +244,18 @@
         "portraitImg": "ammarImg"
       }
     ]
+  },
+  "chatBotMessages": {
+    "headerTitle": "Find help",
+    "goToSeekHelpPageText": "Redirecting to search help page",
+    "goToOpportunitiesPageText": "Redirecting to jobs / training opportunities page",
+    "goToBlogPageText": "Redirecting to blog page",
+    "welcomingMessage": "Hello! How can we help you?",
```

Comment:
welcomeMessage

welcoming would be like "he is a welcoming person", although technically this message is also a welcoming message

Reply:
was just going to fix it :D looked weird to me too